tests/phpunit/testcases/test_template_helper_functions.php

```diff
@@ -189,4 +189,92 @@ function test_prepare_original_with_line_breaks_and_tabs() {
 		$this->assertEquals( $orig, $expected_result );
 	}
 
+	/**
+	 * Expects matching glossary entries preceded and succeded by placeholders.
+	 */
+	function test_map_glossary_entries_to_translation_originals_with_entries_bounded_by_placeholders() {
```

Comment:
Can you refactor this test to make use of a [data provider](https://docs.phpunit.de/en/9.6/writing-tests-for-phpunit.html#data-providers)? This will make it easier to understand what is tested and what may fail in the future.

For inspiration feel free to [take a look at some of WordPress' tests](https://github.com/search?q=repo%3AWordPress%2Fwordpress-develop+%40dataProvider+path%3A%2F%5Etests%5C%2Fphpunit%5C%2Ftests%5C%2F%2F&type=code).

Reply:
Thanks for the suggestion, I'm using it already :)

Reply:
@ocean90 I've refactored to use a data provider. Actually added the test cases to an existent test with the same scope.